app/web/features/profile/view/UserOverview.tsx

```diff
@@ -43,25 +48,79 @@ const useStyles = makeStyles((theme) => ({
       marginBottom: theme.spacing(0.5),
     },
   },
+
+  cardActions: {
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "stretch",
+    padding: theme.spacing(0.5),
+    "& > *": {
+      margin: theme.spacing(0.5),
+    },
+    "& > :not(:first-child)": {
+      marginLeft: theme.spacing(0.5),
+    },
+  },
 }));
 
-export default function UserOverview({ children }: PropsWithChildren<unknown>) {
+// @todo: move this into /components and decouple it from features/profile because it's used
+//        from the dashboard as well
+export default function UserOverview({
+  actions,
+  showHostAndMeetAvailability = true,
+}: PropsWithChildren<{
+  actions?: React.ReactNode;
+  showHostAndMeetAvailability?: boolean;
+}>) {
```

Comment:
```suggestion
}: {
  actions?: React.ReactNode;
  showHostAndMeetAvailability?: boolean;
}) {
```
Since this component is no longer using `children` to render stuff? Or better still, pull that into `UserOverviewProps` so it's easier to read in autocompletes when hovering over the component's usage elsewhere

Reply:
Ah good point! On it